tests/test_module_doc.py

```diff
@@ -26,9 +26,7 @@
 
 # Ignored items will not have their default values checked or be included for
 # alphabetical order purposes
-IGNORE_ITEM = [
-    ('usbguard', 'format_notification_message')  # newlines breaks docstring
-]
+IGNORE_ITEM = [("usbguard", "format_notification_message")]  # newlines breaks docstring
```

Comment:
is it possible to fix those or this is permanent ignore list?

Reply:
@ultrabug 
```diff
diff --git a/py3status/modules/usbguard.py b/py3status/modules/usbguard.py
index 060409f..6c85b86 100644
--- a/py3status/modules/usbguard.py
+++ b/py3status/modules/usbguard.py
@@ -35,7 +35,7 @@ Configuration parameters:
         '[ {format_button_allow}][ {format_button_block}]')*
     format_device_separator: show separator if more than one (default ' ')
     format_notification_message: specify notification message to use
-        *(default 'USB ID: {usb_id}\nName: {name}\nPort: {port}')*
+        (default 'USB ID: {usb_id}\\nName: {name}\\nPort: {port}')
     format_notification_title: specify notification title to use
         *(default 'USB Device [\?if=policy=allow Allowed]'
         '[\?if=policy=block Blocked][\?if=policy=reject Rejected]')*
diff --git a/tests/test_module_doc.py b/tests/test_module_doc.py
index d7610ca..54fc6b5 100644
--- a/tests/test_module_doc.py
+++ b/tests/test_module_doc.py
@@ -26,9 +26,7 @@ IGNORE_ILLEGAL_CONFIG_OPTIONS = [("group", "align")]
 
 # Ignored items will not have their default values checked or be included for
 # alphabetical order purposes
-IGNORE_ITEM = [
-    ('usbguard', 'format_notification_message')  # newlines breaks docstring
-]
+IGNORE_ITEM = []
 
 # Obsolete parameters will not have alphabetical order checked
 OBSOLETE_PARAM = []
````

Reply:
I believe the PR contains the fix no? I'm lost here

Reply:
Yeah, with your PR and lasers' patch we will be fully blacked with zero exceptions to the rule 😉

Reply:
Then change & merge at will guys!